set.mm

```diff
@@ -62783,17 +62788,33 @@ ordered pairs (for use in defining operations).  This is a special case
 
   ${
     fvoveq1d.1 $e |- ( ph -> A = B ) $.
-    $( Equality deduction for nested function and operation value.
+    $( Equality deduction for nested function and operation value.  Technical
+       theorem to be used to reduce the size of a significant number of proofs.
        (Contributed by AV, 23-Jul-2022.) $)
     fvoveq1d $p |- ( ph -> ( F ` ( A O C ) ) = ( F ` ( B O C ) ) ) $=
       ( co oveq1d fveq2d ) ABDFHCDFHEABCDFGIJ $.
   $}
 
   $( Equality theorem for nested function and operation value.  Closed form of
-     ~ fvoveq1d .  (Contributed by AV, 23-Jul-2022.) $)
+     ~ fvoveq1d .  Technical theorem to be used to reduce the size of a
+     significant number of proofs.  (Proposed by @SAVASK, 8-Jul-2022)
```

Comment:
The Proposed label is not exactly a novelty (see elpreqprb), but still rare.  I occasionally attributed a proof idea to someone else in the comment (as in aevlem), if its complexity/ingenuity called for it.  But in this particular case not even the proof is meant...  Will this really create extra motivation for similar suggestions?

We agreed that the number of labels is in excess, and I changed my habits accordingly.  This seems to be contrary to this idea.  What do you 8others) think?

Reply:
I think it's nice and "honest" to mention *significant* help from other people in a formalization, like @avekens, @jkingdon and you (@wlammen) did by mentioning me on a few occasions.  (I think @jkingdon and @avekens have sometimes been "too" nice by mentioning me when I just said "you could prove that" without giving details on the proof !).  As for the way to mention it, I have no real opinion and I'm fine with the various existing ones.

What do you mean by "the number of labels is in excess" ?

Reply:
Since the Proposal labels are (and will be) rare, I do not see the danger/risk to clutter the comments with too many parentheticals of this kind.
I would leave it to the contributor to decide if he/she wants to mention another one who was involved (and to decide if the help was significant or not). Sometimes it is helpful just to raise a vague idea to formulate a theorem (and then to prove it). 

To provide this information in a semi-formal way (as parentheticals) has the advantage that it is available in a uniform way (e.g. also to seach for it).

By the was, I often got (and get) hints by Gérard Lang for new/additianal proofs, and I really would like to document in set.mm that this was his idea.

Reply:
PS: If someone explicitly does not want to mentioned (like @savask in this case), this should be respected of course.

Reply:
The current Proposal labels are removed now from this PR (as @savask asked for). Therefore, the discussion about this issue should be continued elsewhere (if necessary).